cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -36,22 +37,40 @@
 
 # storage_prefix = config.get("runtime.cloud.storage_prefix")
 
+
 runtime = runtimes.get_runtime()
 
-intermediate_cache_store = CacheStore(
-    config.get("storage.intermediate.uri"),
-    storage_options=config.get("storage.intermediate.storage_options"),
-)
 
+# Transform Functions into Tasks -----------------------------------------------------------
 
-results_cache_store = CacheStore(
-    config.get("storage.results.uri"),
-    storage_options=config.get("storage.results.storage_options"),
-)
 
-# Transform Functions into Tasks -----------------------------------------------------------
+@task()
```

Comment:
If the fact that these are coming in as `Parameter`s and not raw values, we may want to try using Prefect's `context` instead.